daml-assistant/integration-tests/src/DA/Daml/Assistant/CreateDamlAppTests.hs

```diff
@@ -95,9 +95,13 @@ tests =
           step "Install dependencies for UI"
           createDirectoryIfMissing True "node_modules"
           cachedDeps <- locateRunfiles (mainWorkspace </> "daml-assistant" </> "integration-tests" </> "create_daml_app_deps.tar")
-          runConduitRes
-              $ sourceFileBS cachedDeps
-              .| Tar.Conduit.Extra.untar (Tar.Conduit.Extra.restoreFile (\a b -> fail (T.unpack $ a <> " " <> b)) ".")
+          -- This crashes when updating NodeJS and I don't have time or
+          -- interest to debug it.
+          --runConduitRes
+          --    $ sourceFileBS t grep sourceFileBS
+          --    cachedDeps
+          --    .| Tar.Conduit.Extra.untar (Tar.Conduit.Extra.restoreFile (\a b -> fail (T.unpack $ a <> " " <> b)) ".")
+          callCommandSilent ("tar --strip-components=1 -x -f " <> cachedDeps)
```

Comment:
Should we create an issue for this? Or simply leave it at that and remove the comments?

Reply:
I really don't care either way. I lef it there in case anyone really cares about extracting tarballs in-process for some reason I'm not aware of.